submissions/description/server_security_misconfiguration/no_rate_limiting_on_form/change_password/template.md

```diff
@@ -1,24 +1,24 @@
-# No Rate Limiting on Change Password Form
+# No Rate Limiting on Password Change Form
 
-## Overview
+## Overview of the Vulnerability
 
-Rate Limiting prevents an application from becoming unresponsive or unavailable due to too many requests exhausting the application's resources. Without rate limiting on the change password form, a malicious attacker is able to brute force passwords and potential gain access to legitimate users' accounts.
+Rate Limiting prevents an application from becoming unresponsive or unavailable due to too many requests exhausting the application's resources. A lack of rate limiting on a password changeform allows an attacker to send a number of requests to the server which will attempt to change the password of a user with the supplied credentials, even if they are invalid. This has the risk of an attacker being able to attempt to bruteforce credentials for users without any protections, and can be used as a method of credential stuffing and compromising accounts to the service.
```

Comment:
Typo, missing space between 'change' and 'form'

Reply:
remediated, ty